src/app/components/pages/wallets/address-detail/wallet-detail.component.scss

```diff
@@ -123,3 +127,66 @@
     }
   }
 }
+
+@keyframes floatup {
+  20% {
+    opacity: .999
+  }
+
+  100% {
+    -webkit-transform: translate3d(-50%, -17px, 0);
+    transform: translate3d(-50%, -17px, 0)
+  }
+}
+
+@-webkit-keyframes floatup {
+  20% {
+    opacity: .999
+  }
+
+  100% {
+    -webkit-transform: translate3d(-50%, -17px, 0);
+    transform: translate3d(-50%, -17px, 0)
+  }
+}
+
+.click-to-copy {
+  cursor: pointer;
+}
+
+.copy-label {
+  color: #0072ff;
+  display: none;
+  font-size: 12px;
+  line-height: 100%;
+  position: relative;
+  opacity: .999;
+  transition: opacity .2s ease-in-out;
+  top: -1px;
+  padding-left: 10px;
+
+  &.hidden{
+    opacity: .001;
+  }
+
+  &::after {
+    content: attr(data-label);
+    color: #0072ff;
+    display: inline-block;
+    position: absolute;
+    top: -2px;
+    left: 50%;
+    opacity: .001;
+    text-align: center;
+    -webkit-transform: translate3d(-50%,0,0);
```

Comment:
This line is not necessary, the Sass compiler is responsible for placing the prefix.

Reply:
Fixed.